web-js/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
Would it be worth it to create a slick logo for this?

Reply:
Not sure how big of a use case there'll be for saving to an iOS homescreen but we could definitely use a nice favicon and then reuse it as the app icon. I'll ask design to see what's possible.